### ISSUE TITLE
Latest Forge 1.19.2/3 jar doesn't have fix deployed

### ISSUE THREAD

**Issue by AmmoniumX** [alkyaly/Infinity-Water-Bucket#10]
**Tldr**: the current jar for Forge 1.19.2 on curseforge, [Infinity-Water-Bucket-1.19-2.0.0.jar](https://www.curseforge.com/minecraft/mc-mods/infinity-water-bucket/files/3825594), still has code inside `InfinityWaterBucket.java` that tries to access non-existend method `PlayerLoggedInEvent.getPlayer()`. This can and has been verified using a java decompiler such as http://www.javadecompilers.com/ , and causes the game to crash for some players with `java.lang.NoSuchMethodError: 'net.minecraft.world.entity.player.Player net.minecraftforge.event.entity.player.PlayerEvent$PlayerLoggedInEvent.getPlayer()'`. I can see the error was fixed on this commit named "update to 1.19.4", https://github.com/coolsimulations/Infinity-Water-Bucket/commit/eb3daf004a6bcd258090528301c7c97468edbbe5#diff-28354be24a6b5b9b680c7969842b0e66956b36cb796dd506b4b3981f01f25862 , but apparently the jar for 1.19.2/1.19.3 must have been built before that, and were therefore left without said fix.

**Solution**: thankfully the solution should be easy, just build a jar based on the latest state of the codebase, and upload it for 1.19.2/1.19.3

**Crash log**: https://pastebin.com/40AGt0T3

**Comment by coolsimulations** [comment]
Totally fair issue here, I did make a fix for the newer version. Frankly I figured 1.19.2 wasn't going to be used as much as 1.19.4 so not much point in fixing an older version. Anyway, this is old now so will close this issue
